fe/fe-core/src/main/java/org/apache/doris/analysis/AlterTableClause.java

```diff
@@ -40,4 +40,6 @@ public void setTableName(TableName tableName) {
     }
 
     public abstract boolean allowOpMTMV();
+
+    public abstract boolean needChangeMTMVState();
```

Comment:
this may be add a default impl and value such as true, when add new cluse later, maybe safe

Reply:
This is intentional to avoid ignoring this method when creating new classes, and adding some comments in other PRs